apps/docs/content/components/Card/index.mdx

```diff
@@ -0,0 +1,129 @@
+    <Caption>
+      Don't use different image ratios and a different length size for the
+      content.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+Cards typically appear in groups of three or four in desktop viewports. For smaller viewports, or when more than four cards are needed, we recommend stacking the cards vertically. You can use the [Grid](/components/Grid) or the [Stack](/components/Stack) component to achieve this.
+
+<div>
+  <CustomVideoPlayer
+    width="100%"
+    src="https://github.com/primer/brand/assets/912236/e4ac4ed3-4190-4e8b-b467-364fe880aa30"
+  />
+  <Caption>
+    Stack cards vertically so that the content has enough line length.
+  </Caption>
+</div>
+
+## Anatomy
+
+![An image showing the Card anatomy with an icon as a visual asset, a heading, a description and a CTA text in a vertical order.](https://github.com/primer/brand/assets/912236/633b83d8-4ea8-49bd-9d6c-45a1989060d9)
+
+### Visual asset
+
+The visual asset is optional and can be an image or an icon. It can be used to provide a visual cue to the user about the content of the card. To avoid visual noise, we recommend using only one icon or image, but not both in the same card.
+
+![An image showing two Cards, one with an Icon and another with an Image as the visual assets options.](https://github.com/primer/brand/assets/912236/be65583b-f6f0-438a-8d2c-6b7a0a9fd3de)
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/863de864-5216-44bf-8b35-430821da8853" />
+    <Caption>Use only one visual asset per card.</Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/9acdde97-42e3-4a7a-8102-b9be5e3c05fc" />
+    <Caption>Don't use both icon and image as the visual asset.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Label
+
+The label can be used to indicate the type of content or its status. For example, use it to indicate that the content is `new` or that it is a `beta` feature. The label is optional and can be used along with a visual asset.
+
+![An image showing two Cards, one with a Label and another one with Label and Icon.](https://github.com/primer/brand/assets/912236/2d84fe89-2735-451e-9d1f-9a2f0297e7b5)
+
+### Heading
+
+The heading is the main title of the card. It should be short and concise.
+
+### Description
+
+The description is an optional short text that extends the information provided by the heading.
+
+### Call to action
+
+The call to action label indicates the action or resource the card links to. It defaults to "Learn more".
+
+## Related components
+
+- To display non-navigational related content in a compact way, use the [Pillar](/components/Pillar) component.
+- To display a list of plain content items, use the [UnorderedList](/components/UnorderedList) or the [OrderedList](/components/OrderedList) component.
+- To display a single block of content to draw attention or create urgency around a user action, use the [CTABanner](/components/CTABanner) component.
```

Comment:
I love this section. Wondering if we could reorder it for improved scanning by front-loading the component name. E.g.

```suggestion
- [Pillar](/brand/components/Pillar): To display non-navigational related content in a compact way, use the  component.
- [OrderedList](/brand/components/OrderedList): To display a list of plain content items
- [CTABanner](/brand/components/CTABanner): To display a single block of content to draw attention or create urgency around a user action
```

Think you'll need to prefix the links here too btw ☝️

Reply:
Re-order sounds good!